### DISCUSSION TITLE
Can you create a webhook for comment mentions?

### DISCUSSION THREAD

**Discussion by BlakeMScurr** [community/community#22565] [New to GitHub]
<p>I’d like to trigger an action if someone mentions me in a comment. I know about the notifications API, but I’d rather not have to poll too often.</p>

**Comment by mpboom** [comment, marked as answer]
<p>All events are listed right here: <a href="https://developer.github.com/webhooks/#events" rel="nofollow noopener">https://developer.github.com/webhooks/#events</a></p>
